kroxylicious-operator/pom.xml

```diff
@@ -159,6 +159,12 @@
             <artifactId>log4j-slf4j2-impl</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>io.strimzi</groupId>
+            <artifactId>api</artifactId>
+            <version>0.47.0</version>
```

Comment:
we should pull out version number as a constant and pushing it up to the parent.